source/features/pr-filters.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {checkInline} from '../libs/icons';
+import features from '../libs/features';
+import {fetchCIStatus} from './ci-link';
+
+let currentQuerySegments: string[];
+
+function addDropdownItem(dropdown: HTMLElement, title: string, filterCategory: string, filterValue: string): void {
+	const filterQuery = `${filterCategory}:${filterValue}`;
+
+	const isSelected = currentQuerySegments.some(
+		segment => segment.toLowerCase() === filterQuery
+	);
+
+	const query = currentQuerySegments.filter(
+		segment => !segment.startsWith(`${filterCategory}:`)
+	).join(' ');
+
+	const search = new URLSearchParams({
+		q: query + (isSelected ? '' : ` ${filterQuery}`)
+	});
+
+	dropdown.append(
+		<a
+			href={`?${search}`}
+			className={`select-menu-item ${isSelected ? 'selected' : ''}`}
+			aria-checked={isSelected}
+			role="menuitemradio"
+		>
+			{checkInline()}
+			<div className="select-menu-item-text">{title}</div>
+		</a>
+	);
+}
+
+function addDraftFilter(reviewsFilter: HTMLElement): void {
+	reviewsFilter.addEventListener('toggle', () => {
```

Comment:
Can you move `addEventListener` to `init` then the listener function can stay here in the top level.